r2/r2/models/link.py

```diff
@@ -217,6 +217,12 @@ def _submit(cls, is_self, title, content, author, sr, ip,
             url = content
             selftext = cls._defaults["selftext"]
 
+        nsfwcheck = re.compile(r"\bnsf[wl]\b", re.I)
```

Comment:
No need to define another copy of the regex, you can just use `cls._nsfw` instead of this.